src/pages/heeyeolKim/Login/Login.js

```diff
@@ -1,109 +1,105 @@
+            buttonToGo: "buttonOff",
         }
     }
 
-    // handleActivation = () => {
-    //     //const validEmailRegex = RegExp(/^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i);
-    //     if (this.state.idValue.includes("@") && this.state.pwValue.length >= 5) {
+    handleActivation = () => {
+        const { idValue, pwValue } = this.state;
+        let isLoginValid = idValue.includes("@") && pwValue.length >= 5;
+
+        isLoginValid ? this.setState({buttonToGo: "buttonOn"}) : this.setState({buttonToGo: "buttonOff"})
+    }
+
+    // handleInput = (e) => {
+    //     const {type, name, value} = e.target;
+    //     if (type === "text") {
     //         this.setState({
-    //             buttonOn: true,
-    //             buttonOpacity: 1,
-    //         })
-    //     } else {
+    //             idValue: value
+    //         });
+    //     } else if (type === "password") {
     //         this.setState({
-    //             buttonOn: false,
-    //             buttonOpacity: 0.3,
-    //         })
+    //             pwValue: value
+    //         });
     //     }
     // }
 
-    handleActivation = () => {
-        //const validEmailRegex = RegExp(/^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i);
-        ( this.state.idValue.includes("@") && this.state.pwValue.length >= 5 )
-        ? this.setState({
-            buttonToGo: true,
-            buttonOpacity: 1,
-        })
-        : this.setState({
-            buttonToGo: false,
-            buttonOpacity: 0.3,
+    handleInput = e => {
+        const { value, name } = e.target;
+        this.setState({
+            [name]: value
         })
     }
-    
-// disabled, preventDefalut
-
-    handleInput = (e) => {
-        //e.preventDefalut();
-        if (e.target.type === "text") {
-            this.setState({
-                idValue: e.target.value
-            });
-        } else if (e.target.type === "password") {
-            this.setState({
-                pwValue: e.target.value
-            });
-        }
-    }
 
     goToMain = () => {
-        if (this.state.buttonToGo === true) {
-            this.props.history.push('/main-heeyeol') // 정확히 어떤 의미?  
-        } // 아니면 그냥 초기값(false이므로 else를 작성 안해도 됨)
+        fetch('http://10.58.5.70:8000/user/login', 
+        {method: 'POST', 
+        body: JSON.stringify({
+            email: this.state.idValue,
+            password: this.state.pwValue,
+        })
+        })
+        .then((res) => {
+            console.log(res);
+            return res.json();})
+        .then((data) => {
+            console.log("결과: ", data);
+            if (data.MESSAGE === "SUCCESS") {
+                this.props.history.push('/main-heeyeol')
+                } else {
+                    alert("실패!");
+                }
```

Comment:
```suggestion
            if (data.MESSAGE === "SUCCESS") {
                this.props.history.push('/main-heeyeol')
             } else {
                alert("실패!");
             }
```
- 항상 가독성을 위해 들여쓰기 신경써주세요!